spec/index.bs

```diff
@@ -1096,14 +1104,16 @@ Note:
 {{GPUBuffer}} has been created.
 
 <div class=note>
-    {{GPUBuffer}} has a state machine where the states are:
-
-     - [=buffer state/unmapped=] and [=buffer state/destroyed=] with a `null`
-        {{GPUBuffer/[[mapping]]}}
-     - [=buffer state/mapped for reading=] and [=buffer state/mapped for writing=]
-        with an {{ArrayBuffer}} typed {{GPUBuffer/[[mapping]]}}.
-     - [=buffer state/mapping pending for reading=] and [=buffer state/mapping pending for writing=]
-        with a {{Promise}} typed {{GPUBuffer/[[mapping]]}}.
+   Note: {{GPUBuffer}} has a state machine with the following states.
+    ({{GPUBuffer/[[mapping]]}}, {{GPUBuffer/[[mapping_range]]}},
+    and {{GPUBuffer/[[mapped_ranges]]}} are null when not specified.)
+
+     - [=buffer state/unmapped=] and [=buffer state/destroyed=].
+     - [=buffer state/mapped=] or [=buffer state/mapped at creation=] with an
+        {{ArrayBuffer}} typed {{GPUBuffer/[[mapping]]}}, a sequence of two
```

Comment:
nit: oh, looks like the [[mapping]] definition is right; it's here where it doesn't mention promise

Reply:
Yes because it is a promise only in the `[=buffer state/mapping pending=]`. See last bullet point.